src/app/modules/courses/components/course-card/course-card.component.html

```diff
@@ -16,8 +16,8 @@ <h2>
       {{course.description}}
     </div>
     <div class="action">
-      <button (click)="edit.emit()">Edit</button>
-      <button (click)="delete.emit()">Delete</button>
+      <button (click)="edit.emit(course.id)">Edit</button>
```

Comment:
From my experience it is much better to emit the whole course object. In this case you will be able to use data from this object in order to prefill edit data from (in next MR)